app/views/refinery/inquiries/admin/inquiries/_inquiry.html.erb

```diff
@@ -4,26 +4,27 @@
     <span class="preview"><%= truncate(strip_tags(sanitize(inquiry.message)), :length => 60) -%></span>
   </span>
   <span class='actions'>
-    <%= link_to refinery_icon_tag('delete.png'), 
-                refinery.inquiries_admin_inquiry_path(inquiry),
-                :class => "cancel confirm-delete",
-                :title => t('refinery.inquiries.admin.inquiries.delete'),
-                :data => {
-                  :confirm => t('refinery.admin.delete.message', :title => inquiry.name)
-                },
-                :method => :delete -%>
+    <%= action_icon :delete,
+                    refinery.inquiries_admin_inquiry_path(inquiry),
+                    t('refinery.inquiries.admin.inquiries.delete'),
+                    :class => "cancel confirm-delete",
+                    :data => {
+                        :confirm => t('refinery.admin.delete.message', :title => inquiry.name)
+                    },
+                    :method => :delete -%>
 
-    <%= link_to refinery_icon_tag('zoom.png'), refinery.inquiries_admin_inquiry_path(inquiry),
-                :title => t('.read_inquiry') -%>
+      <%= action_icon :preview,
```

Comment:
Use this following `action_icon` to prevent a target blank of the `:preview` action_icon:

```ruby
      <%= action_icon :show,
                      refinery.inquiries_admin_inquiry_path(inquiry),
                      t('.read_inquiry'),
                      class: 'preview_icon'
      -%>
```